src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Options/GeneralExternalSettingsProviderService.cs

```diff
@@ -9,15 +9,14 @@
 using System.Runtime.InteropServices;
 using System.Threading;
 using System.Threading.Tasks;
-//using Microsoft.VisualStudio.Text.Editor.Commanding.Commands;
 using Microsoft.VisualStudio.Utilities.UnifiedSettings;
 using NuGet.Configuration;
 using NuGet.VisualStudio;
 
-namespace NuGet.PackageManagement.VisualStudio.Services
+namespace NuGet.PackageManagement.VisualStudio.Options
 {
     [Guid("6C09BBE2-4537-48B4-87D8-01BF5EB75901")]
-    public sealed class ExternalSettingsProviderService : IExternalSettingsProvider
+    public sealed class GeneralExternalSettingsProviderService : IExternalSettingsProvider
```

Comment:
GeneralPage maybe?

General makes me think general as the meaning rather than the general settings page.

Reply:
I'm working on an abstract implementation already, so I'll name these to be a little more page-like as part of that refactoring.

Reply:
Are you gonna push that refactoring in this PR or a different one?

Reply:
Was planning a different one.

Reply:
Feel free to resolve it when you've finalized the plan :D

Reply:
Renamed it now since I was doing other refactoring around it.